python/cudf/cudf/io/orc.py

```diff
@@ -223,6 +223,7 @@ def read_orc(
     skiprows=None,
     num_rows=None,
     use_index=True,
+    decimal_cols_as_float=[],
```

Comment:
Generally you don't want to use an empty list or anything mutable as a default argument in python because when you later mutate that argument inside the function it can 'stick'. See caveats here https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/